pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -57,11 +61,31 @@
  * <p>TODO: For performance concern, use {@code List<Sketch>} as the intermediate result.
  */
 public class DistinctCountThetaSketchAggregationFunction implements AggregationFunction<Map<String, Sketch>, Long> {
+
+  public enum MergeFunction {
+    SET_UNION, SET_INTERSECT, SET_DIFF;
+
+    /** SET_INTERSECT($1, $2, $3) => "SET_INTERSECT($1,$2,$3)", useful for unit tests */
+    public String apply(final String... args) {
+      final String delimited = String.join(",", args);
+      return String.format("%s(%s)", name(), delimited);
+    }
+
+    public static boolean isValid(final String name) {
```

Comment:
(Code convention) We don't use `final` within method argument or local variables. Same for other places